File: AslihanGulseren/gmailApp/Email.ts
export class Email{
  id:number;
  status:number;
  time:Date;
  senderId:number;
  receiverId:number;
  constructor(){
    this.id=1;
    this.status=1;
    this.time=new Date(1478708162000);
    this.senderId=1;
    this.receiverId=1;
  }
}

File: AslihanGulseren/mailapi/Models/Email.cs
namespace mailapi.Models
{
    public class Email
    {
        public int Id { get; set; }
        public int Status { get; set; }
        public System.DateTime Time { get; set; }
        public int SenderId { get; set; }
        public int ReceiverId { get; set; }
    }
}